shot-android/src/main/java/com/karumi/shot/FocusCleaner.kt

```diff
@@ -0,0 +1,32 @@
+package com.karumi.shot
+
+import android.view.View
+
+interface FocusCleaner {
+
+    sealed class FocusViewIdentifier {
+        class FocusById(val id: Int?) : FocusViewIdentifier()
+        class FocusByTag(val tag: Any?) : FocusViewIdentifier()
```

Comment:
Was thinking about adding the possibility to focus a view by its tag, since in some cases, a view tree contains several views with the same id (e.g. in recycler views)

On the other hand one can set  unique tags dynamically after inflating a view or before taking the screenshot.

I think this could also be a good idea for `disableFlakyViews()`